src/material-examples/table-selection/table-selection-example.html

```diff
@@ -5,13 +5,15 @@
     <th mat-header-cell *matHeaderCellDef>
       <mat-checkbox (change)="$event ? masterToggle() : null"
                     [checked]="selection.hasValue() && isAllSelected()"
-                    [indeterminate]="selection.hasValue() && !isAllSelected()">
+                    [indeterminate]="selection.hasValue() && !isAllSelected()"
+                    [aria-label]='checkboxLabel()'>
```

Comment:
nit: Use double-quote in HTML templates

Reply:
I must've missed it, thanks for pointing it out! It's addressed in the latest commit.